bentoml/service/inference_api.py

```diff
@@ -168,6 +168,26 @@ def user_func(self):
         @functools.wraps(self._user_func)
         def wrapped_func(*args, **kwargs):
             def handle_error(tasks, status_code: int, error_message: str, exception: Exception):
+                def log_request():
```

Comment:
Maybe extract it to an outer function if you can, just so that reading the `handle_error` would be easier